spec/component/importers/activity_insight_importer_spec.rb

```diff
@@ -4935,19 +4942,30 @@
                                          source_identifier: '92747188475').publication
           p3 = PublicationImport.find_by(source: 'Activity Insight',
                                          source_identifier: '190707482930').publication
+          p4 = PublicationImport.find_by(source: 'Activity Insight',
+                                         source_identifier: '171620739090').publication
 
-          expect(ActivityInsightOaFile.count).to be 3
+          expect(ActivityInsightOaFile.count).to be 4
           expect(p1.activity_insight_oa_files.first.location).to eq('abc123/intellcont/file.pdf')
           expect(p2.activity_insight_oa_files.first.location).to eq('abc123/intellcont/file-3.pdf')
           expect(p3.activity_insight_oa_files.first.location).to eq('abc123/intellcont/file-5.pdf')
+          expect(p4.activity_insight_oa_files.first.location).to eq('abc123/intellcont/file-6.pdf')
         end
 
-        it 'does not import ActivityInsightOaFiles for imported publications without postprint/open access file locations' do
+        it 'verifies DOI for those publications' do
           importer.call
           p4 = PublicationImport.find_by(source: 'Activity Insight',
+                                         source_identifier: '171620739090').publication
+          expect(p4.doi_verified).to be true
```

Comment:
I'm not too sure we should be testing how the DOIVerificationJob works in this test since it is an async job.  Perhaps we should test that the job is queued or that `DoiVerificationJob.new.perform(pub_record)` was invoked instead.

Reply:
I think it may be prone to breaking if we rely on the outcome of an async job.

Reply:
Yeah, we might end up just removing the job queuing from the importer though, right?